nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/NonDexMojo.java

```diff
@@ -59,33 +59,33 @@ a copy of this software and associated documentation files (the
 public class NonDexMojo extends AbstractNonDexMojo {
 
     private List<NonDexSurefireExecution> executions = new LinkedList<>();
+    private MojoExecutionException allExceptions = null;
```

Comment:
Why did you move this to a field? It doesn't seem to be something that needs to reside in a field.